src/components/Image/CombinedRoundedImages.tsx

```diff
@@ -11,13 +11,13 @@ const CombinedImagesWrapper = styled.div<{ size?: number }>`
   margin-top: 2px;
 `;
 
-const SmallImageWrapper = styled.div<{ size?: number }>`
+const SmallImageWrapper = styled.div<{ size?: number; smallBgColor?: string }>`
```

Comment:
let's rename it to something more common like borderColour since `smallBgColor` is only single scenario case, if u wanna use this way then make it boolean and put colour in css below, but I would recommend renaming it as `borderColour` 
```suggestion`
const SmallImageWrapper = styled.div<{ size?: number; smallBgColor?: string }>`
```

Reply:
Yeah you're right borderColor makes more sense